Libraries/Components/Keyboard/KeyboardAvoidingView.js

```diff
@@ -13,6 +13,7 @@
 
 const Keyboard = require('Keyboard');
 const LayoutAnimation = require('LayoutAnimation');
+const NativeMethodsMixin = require('react/lib/NativeMethodsMixin');
```

Comment:
Oh, this changed recently in 3683beb88ab3233038ac32b9c33112d537478088. Addressed in b3485a8.